EXILED/Exiled.API/Features/Player.cs

```diff
@@ -145,6 +142,16 @@ public Player(GameObject gameObject)
         /// </summary>
         public static Dictionary<string, Player> UserIdsCache { get; } = new(20);
 
+        /// <summary>
+        /// Gets or sets a <see cref="CustomHealthStat"/>.
+        /// </summary>
+        public CustomHealthStat CustomHealthStat { get; protected set; }
+
+        /// <summary>
+        /// Gets or sets a <see cref="CustomHumeShieldStat"/>.
+        /// </summary>
+        public CustomHumeShieldStat CustomHumeShieldStat { get; protected set; }
```

Comment:
I have a problem making it public now make me a breaking change for lab api

Reply:
What?

Reply:
CustomHumeShieldStat (not complettelly for this one since it's still can't be modify for on of it's aspect but MaxHS is now possible without it)